### DISCUSSION TITLE
Reading customized properties from Config File

### DISCUSSION THREAD

**Discussion by Bajibabup** [webdriverio/codemod#92] [General]
Hi,
I need to define variables/properties in wdio.config.js, and in-turn need to access/read in spec file.
Is there any best way to do it?
I tried below approach
1.  Defined the variables/object in wdio config file.
2. tried to use browser.options(as I am using WDIO V8) to get that value. But, it not working. 
 getting message like concerned variable/property is undefined"

**Code in Config file:**
exports.config = {
 testURL1:'https://webdriver.io/'
const login = {
  'ID': 'run1',
  'Pass': 'pass',
  'Url' : 'https://aaa.com',
  }; 
}

**Spec file code:**

describe('tests',async()=>
{
  
   
    it(' Page TC',async()=>
{
 console.log("Env custom Paramater1:", browser.options. testURL1) //not working, message like Env custom Paramater1: undefined
     console.log("Env custom Paramater2:", browser.options.login) ///not working, message like Env custom Paramater2: undefined
})

})

**Comment by christian-bromann** [comment]
> I need to define variables/properties in wdio.config.js

The WebdriverIO config file is not the place for that, I recommend define a different file for it.
